src/helpers/LinkComponent.types.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
I _think_ you should be able to make these `.d.ts` files instead of `.types.ts` file and import them via their basename (i.e. `import { LinkComponentProps } from "./LinkComponent"`).